Packs/RsaNetWitnessPacketsAndLogs/Integrations/RSANetWitnessPacketsAndLogs/RSANetWitnessPacketsAndLogs.yml

```diff
@@ -169,34 +184,55 @@ script:
-        (#-#) whose packets will be streamed back.
+      description: <string> A comma separated list of session ids or session id ranges (#-#) whose packets will be streamed back.
     - name: packets
-      description: <string> A comma separated list of packet ids or session/packet
-        ids (#&#) that will be streamed back.
+      description: <string> A comma separated list of packet ids or session/packet ids (#&#) that will be streamed back.
     - name: where
-      description: <string> Where clause which will be evaluated to determine which
-        sessions to stream back
+      description: <string> Where clause which will be evaluated to determine which sessions to stream back
     - name: time1
-      description: <date-time> A starting time range (UTC) where matching packets
-        will be streamed back ("2010-Apr-20 09:00:00")
+      description: <date-time> A starting time range (UTC) where matching packets will be streamed back ("2010-Apr-20 09:00:00")
     - name: time2
-      description: <date-time> An ending time range (UTC) where matching packets will
-        be streamed back ("2010-Apr-20 10:00:00")
+      description: <date-time> An ending time range (UTC) where matching packets will be streamed back ("2010-Apr-20 10:00:00")
     - name: flags
       description: <uint32> Additional flags as defined by the NwPackets SDK function
     - name: concentratorIP
       description: <string> Use the instance that is mapped to this concentrator address
     - name: concentratorPort
-      description: <uint32> Port to use with concentratorIP, 50105 for Non SSL and
-        56105 for SSL
+      auto: PREDEFINED
+      predefined:
+      - "50102"
+      - "50103"
+      - "50104"
+      - "50105"
+      description: Port to use with concentratorIP, usually 50105 but that can be changed by setting the /rest/config/port parameter
+      defaultValue: "50105"
+    - name: useSSL
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Send request using SSL. To enable it in your RSA appliance, go to the setting in /sys/config/ssl
+      defaultValue: "true"
     - name: render
-      description: <string> The render type - pcap, logs, logs (csv), logs (xml) or
-        logs (json)
+      auto: PREDEFINED
+      predefined:
+      - text/plain
+      - text/csv
+      - text/xml
+      - application/json
+      description: The render type - pcap, logs, logs (csv), logs (xml) or logs (json)
+    - name: noFile
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: If noFile is set to True and render to application/json, this command will not create a file and instead output everything in the contextdata
+    outputs:
+    - contextPath: NetWitness.Packets
+      description: Raw logs, only if noFile option is set
     description: Stream packets back based on the input parameters provided
```

Comment:
```suggestion
    - name: renderToContext
      defaultValue: "false"
      auto: PREDEFINED
      predefined:
      - "true"
      - "false"
      description: |
    If renderToContext is set to True and the render type is set to logs (json),
    the command will not create a file with the output, it will output to the context data.
    outputs:
    - contextPath: NetWitness.Packets
      description: Raw logs. This output is only available if the renderToContext option is set to true.
    description: Stream packets back based on the input parameters provided
```